src/test/java/org/dinosaur/foodbowl/domain/comment/docs/CommentControllerDocsTest.java

```diff
@@ -37,28 +43,60 @@ public class CommentControllerDocsTest extends MockApiTest {
-                                        fieldWithPath("message").type(JsonFieldType.STRING).description("댓글 내용")
-                                )
-                        )
-                );
+        private final String token = jwtTokenProvider.createAccessToken(1L, RoleType.ROLE_회원);
+
+        @Test
+        @DisplayName("등록을 문서화한다.")
+        void createComment() throws Exception {
+            given(commentService.save(anyLong(), any())).willReturn(1L);
+
+            mockMvc.perform(post("/api/v1/comments")
+                            .header("Authorization", "Bearer " + token)
+                            .content(objectMapper.writeValueAsString(new CommentCreateRequest(1L, "다즐 너무 멋져")))
+                            .contentType(MediaType.APPLICATION_JSON)
+                            .characterEncoding(StandardCharsets.UTF_8))
+                    .andExpect(status().isCreated())
+                    .andExpect(header().string("Location", "/api/v1/posts/1"))
+                    .andDo(document("api-v1-comments-create",
+                                    requestHeaders(
+                                            headerWithName(HttpHeaders.AUTHORIZATION).description("서버에서 발급한 엑세스 토큰")
+                                    ),
+                                    requestFields(
+                                            fieldWithPath("postId").type(JsonFieldType.NUMBER).description("게시글 ID"),
+                                            fieldWithPath("message").type(JsonFieldType.STRING).description("댓글 내용")
+                                    )
+                            )
+                    );
+        }
+
+        @Test
+        @DisplayName("수정을 문서화한다.")
+        void updateComment() throws Exception {
+            willDoNothing().given(commentService).updateComment(anyLong(), anyLong(), any(CommentUpdateRequest.class));
+
+            mockMvc.perform(put("/api/v1/comments/{commentId}", 1L)
+                            .header("Authorization", "Bearer " + token)
+                            .content(objectMapper.writeValueAsString(new CommentUpdateRequest("안녕하세요")))
+                            .contentType(MediaType.APPLICATION_JSON)
+                            .characterEncoding(StandardCharsets.UTF_8))
+                    .andExpect(status().isNoContent())
+                    .andDo(document("api-v1-comments-update",
+                                    requestHeaders(
+                                            headerWithName(HttpHeaders.AUTHORIZATION).description("서버에서 발급한 엑세스 토큰")
+                                    ),
+                                    pathParameters(
+                                            parameterWithName("commentId").description("댓글 ID")
+                                    ),
+                                    requestFields(
+                                            fieldWithPath("message").type(JsonFieldType.STRING).description("댓글 내용")
```

Comment:
```suggestion
                                            fieldWithPath("message").type(JsonFieldType.STRING).description("수정할 댓글 내용")
```
수정할 댓글 내용이라는 것을 명시적으로 적어주는건 어떨까요? :)